test/Microsoft.AspNet.StaticFiles.Tests/StaticFileMiddlewareTests.cs

```diff
@@ -31,6 +31,18 @@ public class StaticFileMiddlewareTests
             Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
         }
 
+        [Fact]
+        public async Task InvalidCharacters_PassesThrough()
+        {
+            TestServer server = StaticFilesTestServer.Create(app => app.UseStaticFiles(new StaticFileOptions()
+            {
+                RequestPath = new PathString(""),
+                FileProvider = new PhysicalFileProvider(Path.Combine(Directory.GetCurrentDirectory(), "."))
```

Comment:
It is a piece of silly code though. I was thinking "someone could turn this into a theory, add inline data, and replace the '.'". In the end it just looks like I'm jumping through hoops for nothing.

Would you like me to change it now or ket it ride through?